integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -85,19 +82,25 @@ object HiveExample {
 
     logger.info(s"============HIVE CLI IS STARTED ON PORT $port ==============")
 
-    statement.execute("CREATE TABLE IF NOT EXISTS " + "HIVE_CARBON_EXAMPLE " +
-                      " (ID int, NAME string,SALARY double)")
+    statement.execute("DROP TABLE IF EXISTS HIVE_CARBON_EXAMPLE")
+
+    statement.execute("CREATE TABLE HIVE_CARBON_EXAMPLE " +
+      " (ID int, NAME string,SALARY double) " +
+      "ROW FORMAT SERDE 'org.apache.carbondata.hive.CarbonHiveSerDe' " +
+      "WITH SERDEPROPERTIES ('mapreduce.input.carboninputformat.databaseName'='default', " +
+      "'mapreduce.input.carboninputformat.tableName'='HIVE_CARBON_EXAMPLE')")
+
     statement
       .execute(
         "ALTER TABLE HIVE_CARBON_EXAMPLE SET FILEFORMAT INPUTFORMAT \"org.apache.carbondata." +
-        "hive.MapredCarbonInputFormat\"OUTPUTFORMAT \"org.apache.carbondata.hive." +
-        "MapredCarbonOutputFormat\"SERDE \"org.apache.carbondata.hive." +
-        "CarbonHiveSerDe\" ")
+          "hive.MapredCarbonInputFormat\"OUTPUTFORMAT \"org.apache.carbondata.hive." +
```

Comment:
Can you add space before OUTPUTFORMAT?

Reply:
I have already added space before OUTPUTFORMAT

Reply:
ok